### PR TITLE
TELCODOCS-1150: nmstateconfig update for day2 worker on DHCP subnet

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13 

Issue: https://issues.redhat.com/browse/TELCODOCS-1150

Link to docs preview:
* https://56994--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-manual-install.html#ztp-installation-crs_ztp-manual-install
* https://56994--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes.html#sno-adding-worker-nodes-to-single-node-clusters-manually_add-workers

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc [hunk 1]
```diff
@@ -193,7 +193,7 @@ spec:
 +
 [IMPORTANT]
 ====
-The `NMStateConfig` CR is required for successful worker node deployments. The cluster network DHCP does not automatically set these network settings for the new worker node.
+The `NMStateConfig` CR is required for successful deployments of worker nodes with static IPs. The cluster network DHCP does not automatically set these network settings for the new worker node.
```
[the remark this answers is not in the document]
Comment (amolnar-rh) [reply modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc]: @achuzhoy Updated, PTAL

File: modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc [hunk 2]
```diff
@@ -164,6 +164,38 @@ where:
...
+        ipv6:
+          enabled: false
+  interfaces:
+    - name: "eth0"
+      macAddress: "AA:BB:CC:DD:EE:11"
+----
++
+[IMPORTANT]
+====
+The `NMStateConfig` CR is required for successful deployments of worker nodes with static IPs and for adding a worker node with dynamic IP if the {sno} was deployed with static IP. The cluster network DHCP does not automatically set these network settings for the new worker node.
```
Comment (tmalove) [inline modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc:196]:
```suggestion
The `NMStateConfig` CR is required for successful deployments of worker nodes with static IP addresses and for adding a worker node with a dynamic IP address if the {sno} was deployed with a static IP address. The cluster network DHCP does not automatically set these network settings for the new worker node.
```
See the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_i
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

**Review by tmalove — APPROVED** [review]

**Pull request merged**